### DISCUSSION TITLE
minify css output

### DISCUSSION THREAD

**Discussion by some0ne2** [tailwindlabs/tailwindcss#10066] [Help] [answered]
When browsing the [docs](https://tailwindcss.com/docs/optimizing-for-production) it says:

> For the smallest possible production build, we recommend minifying your CSS with a tool like [cssnano](https://cssnano.co/), and compressing your CSS with [Brotli](https://en.wikipedia.org/wiki/Brotli).

Is there any advantage to use PostCSS plugin regarding the output size or is the output of `npx tailwindcss -o build.css --minify` already minified to the smallest level?

Thank you community.

**Comment by wongjn** [comment, marked as answer]
The `--minify` flag will use cssnano so it would be equivalent:

https://github.com/tailwindlabs/tailwindcss/blob/25d17db78c7c04c0314f8597138dc4564fc07c16/src/cli/build/plugin.js#L304
